src/games/strategy/triplea/ai/proAI/util/ProBattleUtils.java

```diff
@@ -66,10 +66,9 @@ public static boolean checkForOverwhelmingWin(final PlayerID player, final Terri
     }
 
     // Determine if enough attack power to win in 1 round
-    final IntegerMap<UnitType> playerCostMap = BattleCalculator.getCostsForTUV(player, data);
     final List<Unit> sortedUnitsList = new ArrayList<Unit>(attackingUnits);
     Collections.sort(sortedUnitsList,
-        new UnitBattleComparator(false, playerCostMap, TerritoryEffectHelper.getEffects(t), data, false, false));
+        new UnitBattleComparator(false, ProData.unitValueMap, TerritoryEffectHelper.getEffects(t), data, false, false));
```

Comment:
Looks like lots of unused parameters in this method call. Perhaps the static value can be tucked into UnitBattleComparator to begin by getting rid of one.